jwst/associations/tests/test_wfss.py

```diff
@@ -36,3 +37,33 @@ def test_niriss_wfss():
         members_by_type = {member["exptype"]: member for member in product["members"]}
         assert members_by_type["sourcecat"]["expname"] == source_cat
         assert members_by_type["segmap"]["expname"] == segmap
+
+
+def test_miri_wfss():
+    """Test association properties for MIRI WFSS"""
+
+    pool = AssociationPool.read(
+        get_pkg_data_filename(
+            "data/jw09505_20251216t202003_pool.csv", package="jwst.associations.tests"
+        )
+    )
+    cmd_args = ["--dry-run", "--D"]
+    results = Main.cli(cmd_args, pool=pool)
+    asns = results.associations
+
+    # Need 3 associations types: image2, spec2, image3
+    assert len(asns) == 9
+    asn_types = [asn["asn_type"] for asn in asns]
+    assert REQUIRED_ASN_TYPES_MIRI == set(asn_types)
+
+    # Arrange associations by type
+    asn_by_type = {asn["asn_type"]: asn for asn in asns}
```

Comment:
This can be simplified?

```suggestion

    # Arrange associations by type
    asn_by_type = {asn["asn_type"]: asn for asn in asns}
    assert REQUIRED_ASN_TYPES_MIRI == set(asn_by_type.keys())
```